src/api/app/models/event/base.rb

```diff
@@ -149,20 +147,6 @@ def payload
       @payload ||= Yajl::Parser.parse(read_attribute(:payload))
     end
 
-    def notify_backend
-      return false if queued
-      self.queued = true
-      save
-
-      return false unless self.class.raw_type
-      # tell the backend to tell the (old) plugins
-      updated_payload = payload
-      updated_payload['time'] = created_at.to_i
-      logger.debug "notify_backend #{self.class.name} #{updated_payload.inspect}"
-      response = Backend::Api::Server.notify_plugin(self.class.raw_type, updated_payload)
```

Comment:
as you don't need the raw_type anymore, you can now remove them from all classes

Reply:
Ah! yes, good catch!

Reply:
done!, please re-review it @coolo